virtcontainers/kata_agent.go

```diff
@@ -1327,6 +1327,27 @@ func (k *kataAgent) resumeContainer(sandbox *Sandbox, c Container) error {
 	return err
 }
 
+func (k *kataAgent) memHotplugByProbe(addr uint64, sizeMB uint32, memorySectionSizeMB uint32) error {
+	// hot-added memory device should be sliced into the size of memory section, which is the basic unit for
+	// memory hotplug
+	numSection := uint64(sizeMB / memorySectionSizeMB)
```

Comment:
How about returning an error if `memorySectionSizeMB` is zero? :)

Reply:
yes,yes,yes.;) definitely need to avoid `divided by zero`, update asap. ;)